packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -61,9 +59,11 @@ const CompleteResetPassword = ({
 
   return (
     <AppLayout>
-      <p className="text-start text-grey-400 text-sm">
-        <FtlMsg id="password-reset-flow-heading">Reset your password</FtlMsg>
-      </p>
+      <FtlMsg id="password-reset-flow-heading">
```

Comment:
Just curious is it preferred to put the dom element inside the FtlMsg?

Reply:
That's the convention we've been using, yes!